db/seeds.rb

```diff
@@ -43,10 +43,8 @@ def make_lang_skills
 end
 
 [:superadmin, :social_worker, :department_manager].each do |role|
-  user = FactoryBot.create(:user, role: role, email: "#{role}#{EMAIL_DOMAIN}",
+  FactoryBot.create(:user, role: role, email: "#{role}#{EMAIL_DOMAIN}",
     password: 'asdfasdf')
-  user.profile.contact.primary_email = "#{role}#{EMAIL_DOMAIN}"
```

Comment:
The Primary email should be set, in order to have views propperly displayed...

Reply:
`primary_email` is set. I checked in the console for users that where created through the seeds and their `primary_email` was set. I could login and change the primary email for superadmin, department_manager, social_worker and volunteer. Could you please provide some info on how I could reproduce the issues you mention?

Reply:
is it the same email as User.email? I assume not

Reply:
`User.email` and `User.profile.contact.primary_email`are the same for `superadmin`, `department_manager` and `social_worker` users.

Reply:
I found out, that its the callback... I was assuming the factory to hit in a random email, and forgot about that callback that updates the email.
Sorry about that!